### ISSUE TITLE
[DropDownList] When closed the aria-expanded attribute is not switching from true to false

### ISSUE THREAD

**Issue by PaskalevStoyan** [telerik/kendo-angular#3706]
**Describe the bug**
When opening DropDownList the aria-expanded attribute is set to true. When closing the component the aria-expanded attribute is staying true, when it should be converted to false.

**To Reproduce**
1. open the following demo in the documentation: https://www.telerik.com/kendo-angular-ui/components/dropdowns/dropdownlist/
2. inspect the HTML of the dropdownlist 
3. open the dropdownlist and the aria-expanded attribute will switch to true
4. close the dropdownlist and the aria-expanded attribute will still stay true 

**Expected behavior**
When closing the dropdownlist component, the aria-expanded attribute have to switch to false

**Comment by ipeshev** [comment]
Fix available in [latest dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by meriturva** [comment]
@ipeshev 
I guess the issue could be closed.
